x/contractmanager/keeper/msg_server_test.go

```diff
@@ -0,0 +1,44 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/neutron-org/neutron/v3/testutil/contractmanager/keeper"
+	"github.com/neutron-org/neutron/v3/x/contractmanager/types"
+)
+
+func TestMsgUpdateParamsValidate(t *testing.T) {
+	k, ctx := keeper.ContractManagerKeeper(t, nil)
+
+	tests := []struct {
```

Comment:
I decided not to add them since this would entail the need to mock methods that will be called after successful validation pass. Also it seems like these cases are covered in already implemented handler tests.
But I can add them if it's needed.